inference-engine/tests/functional/inference_engine/ngraph_reader/gather_tests.cpp

```diff
@@ -4,7 +4,8 @@
 
 #include <string>
 #include "ngraph_reader_tests.hpp"
-TEST_F(NGraphReaderTests, ReadGatherNetwork) {
```

Comment:
It will be better to add new visitor tests: https://github.com/openvinotoolkit/openvino/tree/master/ngraph/test/visitors/op

Reply:
instead of reader tests or additionaly to them?

Reply:
added new visitor tests